conan/cli/commands/audit.py

```diff
@@ -32,6 +43,8 @@ def audit_scan(conan_api: ConanAPI, parser, subparser, *args):
     # TODO: Do we then want to hide it in the --help?
     subparser.add_argument("--build-require", action='store_true', default=False,
                            help='Whether the provided reference is a build-require')
+    subparser.add_argument("-sl", "--severity-level", action="store", default=9.0, type=float,
```

Comment:
I don't know if having a value by default is the best approach, this feels like something you would want to opt in, no?

Reply:
I think severity=9 should be high enough, and sounds like a good default, better this as default and users can opt-out instead?